docs/getting-started/tutorial.md

```diff
@@ -24,7 +24,7 @@ Start by updating our data layer to model information about moons. We don't want
   }
 
 + model Moon {
-+   id    Int    @id
++   id    Int    @id @default(autoincrement())
 +   name  String
 +   world World
 +   world World  @relation(fields: [id], references: [id])
```

Comment:
That doesn't look right? I'm not sure anymore how to handle back-relations

Reply:
The doubling up is wrong, the relation part is correct. Defined as, "it worked for me" (I ran through the heroku tutorial this morning).

Reply:
```
worldId Int
world    World  @relation(fields: [worldId], references: [id])
```

Reply:
Why does `Moon` model have `worldId` field?

Reply:
Thanks, I ran into the same typo just now 😄

I think it works accidentally. fields define the field on `Moon` which describes the related `World` entity. So it selects the world which has the same id as your moon.

worldId seems the way to go, according to [Prisma doc](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations) : `profile` + `profileId`

Reply:
`worldId` is the foreign key, which for now must be explicitly defined in the schema

Reply:
Thanks @Weakky